src/c++/perf_analyzer/client_backend/client_backend.cc

```diff
@@ -30,25 +30,11 @@
 #include "torchserve/torchserve_client_backend.h"
 #include "triton/triton_client_backend.h"
 
```

Comment:
remove this blank line so that all includes are together (except for client_backend.h which always goes first and is separate)